test/promql/tests/recording/exporter_recording_rules_test.yml

```diff
@@ -0,0 +1,25 @@
+evaluation_interval: 1m
+
+rule_files:
+- ../../../../rhobs/recording/exporter_recording_rules.yml
+
+tests:
+- interval: 1m
+  name: DSExporterTest
+  input_series:
```

Comment:
Can you also add a test case in which the `grafana_ds_up` gives no output (e.g. only `_`s)?

Reply:
not relevant anymore